src/components/shared/Alert.tsx

```diff
@@ -37,7 +38,7 @@ export const Alert = ({ alert, unMountAlert }: AlertProps) => {
 
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
-    // would like some transition animations
+    // transition animations?
```

Comment:
i'll see if i can figure something out with [`@keyframes`](https://www.w3schools.com/cssref/css3_pr_animation-keyframes.asp). not sure whether/not we'll need to make use of one of your js variables.

Reply:
```js
    root: {
      // ...
      transform: 'translateY(-100vh)',
      animation: `slide-in-from-top 0.5s forwards`,
    },
```

and then in /src/assets/global.css:
```css
@keyframes slide-in-from-top {
  from {
    transform: translateY(-100vh);
  }
  to {
    transform: translateY(0);
  }
}
```